### ISSUE TITLE
[AutoTBATS,TBATS] Usage example 

### ISSUE THREAD

**Issue by AzulGarza** [Nixtla/statsforecast#789]
### Description

it might be great to have a tutorial on using tbats and autotbats for multiple seasonalities based on the [mstl](https://nixtlaverse.nixtla.io/statsforecast/docs/tutorials/multipleseasonalities.html) one.

**Comment by MMenchero** [comment]
Yes, I'll start working on that. I think this is important because in our initial run, some users have gotten confused about AutoTBATS/TBATS, so a tutorial should help clear this.

**Comment by AzulGarza** [comment]
awesome @MMenchero! thank you :)
